core/src/main/java/nl/nn/adapterframework/doc/InfoBuilderSource.java

```diff
@@ -365,7 +367,9 @@ private static String replaceNameLastPartWithGroupName(String group, String bean
 		try {
 			XmlUtils.parseXml(Misc.resourceToString(ClassUtils.getResourceURL("digester-rules.xml")), digesterRulesParser);
 		} catch (Exception e) {
-			log.error("Could nog parse digester-rules.xml: " + e.getStackTrace());
+			if(log.isErrorEnabled()) {
```

Comment:
Het is niet nodig om ``if(log.isErrorEnabled()) {`` toe te voegen

Reply:
Ik heb dit weggehaald.